layouts/jobs/bis.html

```diff
@@ -6,7 +6,7 @@
   {{ $header := dict
     "role" $role
     "job" .Parent.Params.job_name
-    "title" "Best-in-slot (BIS) Gear Sets"
+    "title" .Params.title
```

Comment:
I think this should just be `.Title` instead of `.Params.title` – this is one of the default frontmatter items in hugo, and becomes a proper variable without needing `.Params`